json-lint/schemas/inventory/adservers/adsense.json

```diff
@@ -69,6 +69,13 @@
       "full-width": {
         "type": ["boolean", "string"],
         "pattern": "^(\\$\\{.+\\})$"
+      },
+      "ad-format": {
+        "enum": [
```

Comment:
Why should be an enum. I am not seeing theses values in the adsense documentation in AMP. Maybe i am not seeing the right place. 
https://github.com/ampproject/amphtml/blob/master/ads/google/adsense.md